core/src/test/java/com/radixdlt/genesis/olympia/converter/REv2GenesisFromOlympiaTest.java

```diff
@@ -135,6 +135,10 @@ public void genesis_from_olympia_should_initialize_correct_xrd_balances() throws
     }
   }
 
+  /**
+   * Deserializes a file captured from Olympia Node's `StateIRSerializationTest` (namely,
+   * `it_should_successfully_construct_and_then_serialize_and_deserialize_the_intermediate_state`).
+   */
```

Comment:
_(note from author)_

⚠️ Important question!
I actually had to adjust the Olympia test a bit to get a valid validatorUrl in here.
**Will real Olympia validators all have URLs set?**
(if not, our Node will crash during Guice instantiation - which makes me think that a better solution is to have `if !valid { passDummyUrlToEngine() }` in our Node)

Reply:
Ok, so together with @LukasGasior1 we think this is a problem that has to be addressed (because otherwise it is too easy for any Olympia validator to crash Babylon Node during migration).

Facts:
1. In Olympia, the URL is de-facto optional (i.e. considered valid when empty).
2. In Olympia, when present, the URL is validated with [a regex](https://github.com/radixdlt/radixdlt/blob/main/radixdlt-engine/src/main/java/com/radixdlt/atom/REFieldSerialization.java#L82) different than Babylon Engine's [regex](https://github.com/radixdlt/radixdlt-scrypto/blob/015521043473efe58b20bf6ae254bfdf74ebdade/radix-engine-interface/src/api/node_modules/metadata/models/url.rs#L13) (e.g. Olympia allows `ftp://` URLs)

We have the following competing ideas on how to fix this:

**A)** Babylon Node could validate the Olympia's URLs using the Babylon Engine's regex - and when not valid, pass a hardcoded `https://www.radixdlt.com/` to the Engine.
   - This is a **minor Node change only**

**B)** Babylon Node could validate the Olympia's URLs using the Babylon Engine's regex - and when not valid, pass `None` to the Engine
   - This is a **minor Node and Engine change** (make validator URL optional)

**C)** Engine could relax the validation of this field: basically, allow any String, including empty, or a very long one (at least as long as allowed by Olympia)
   - This is a **minor Engine change only**

Reply:
After some consultation, I went with option **B)** (please see the latest commit).

Reply:
Yep, (B) sounds right to me.